inc/profiles.php

```diff
@@ -74,3 +74,18 @@ function after_setup_theme() {
 	add_image_size( 'responsive_profile_large', $width, $height, $crop );
 }
 add_action( 'after_setup_theme', __NAMESPACE__ . '\after_setup_theme' );
+
+if ( ! defined( 'BU_PROFILES_PLUGIN_ACTIVE' ) || ! BU_PROFILES_PLUGIN_ACTIVE ) {
+	/**
+	 * Removes the profile template if BU profiles is not active.
+	 *
+	 * @param array $page_templates list of registered page templates.
+	 * @return array $page_templates list of registered page templates.
+	 */
+	function remove_profile_template( $page_templates ) {
```

Comment:
While it is legal to define a function inside an IF statement, I'd recommend moving the function declaration outside of the IF statement for clarity and consistency.